Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -971,6 +970,7 @@ protected virtual void Dispose(bool dispose)
 
             if (dispose)
             {
+                _pluginManager?.Dispose();
```

Comment:
I don't think this change is needed though. We could just suppress this warning because it is disposed with other disposable parts.

Reply:
True, it would be disposed anyway. Still, `_disposableParts` is a mutable collection, so putting the dispose explicitly is a clearer guarantee that it is disposed, which is why the analyzer flagged it. I think a simple fix makes sense over an unnecessary suppression (which could be considered noise).